test-prow-e2e.sh

```diff
@@ -21,4 +21,13 @@ source ./contrib/oc-environment.sh
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_06-rh-operators.configmap.yaml
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_09-rh-operators.catalogsource.yaml
 
+echo "Checking for dist files:"
+while [ ! -f ./frontend/public/dist/index.html ]
+do
+ echo "#"
```

Comment:
Do we want to do something like:

```
Waiting for dist files...
Waiting...
Waiting...
Waiting...
Build complete and dist folder ready...running e2e tests
```
Just curious if `Waiting...` will be more clear than `#`.  I don't have a hugely strong opinion on this. 😄

Reply:
@benjaminapetersen sure that might be more clear

Reply:
We can follow on this to clarify.  If its eliminating flakes, lets get it merged.